content/docs/examples/advanced-gateways/wildcard-egress-hosts/index.md

```diff
@@ -168,58 +235,61 @@ section of the [Configure an Egress Gateway](/docs/examples/advanced-gateways/eg
 
 {{< text bash >}}
-$ kubectl delete serviceentry wikipedia www-wikipedia
+$ kubectl delete serviceentry www-wikipedia
 $ kubectl delete gateway istio-egressgateway
 $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
 $ kubectl delete destinationrule egressgateway-for-wikipedia
 {{< /text >}}
 
-## HTTPS traffic to arbitrary wildcarded domains
+### Wildcard configuration for arbitrary domains
+
+The configuration in the previous section worked because all the _*.wikipedia.org_ sites can
+be served by any one of the _wikipedia.org_ servers. However, this is not always the case.
+For example, you may want to configure egress control for access to more general
+wildcard domains like `*.com` or `*.org`.
 
-The configuration in the previous section works thanks to the fact that all the _*.wikipedia.org_ sites are apparently
-served by each of the _wikipedia.org_ servers. However, this may not always be the case. In many cases you may want to
-configure egress control for HTTPS access to `*.com` or `*.org` domains, or even to `*` (all the domains).
+Configuring HTTPS traffic to arbitrary wildcard domains introduces a challenge for Istio gateways. In the previous section
+you directed the traffic to _www.wikipedia.org_, which was made known to your gateway during configuration.
+The gateway, however, would not know the IP address of any arbitrary host it receives in a request.
+This is due to a limitation of [Envoy](https://www.envoyproxy.io), the proxy used by the default Istio egress gateway.
+Envoy routes traffic either to predefined hosts, predefined IP addresses, or to the original destination IP address of
+the request. In the gateway case, the original destination IP of the request is lost since the request is first routed
+to the egress gateway and its destination IP address is the IP address of the gateway.
 
-Configuring traffic to arbitrary wildcarded domains introduces a challenge for Istio gateways. In the previous section
-you directed the traffic to _www.wikipedia.org_, and this host was known to your gateway during the configuration.
-The gateway, however, cannot know an IP address of an arbitrary host it receives a request for. Would you want to
-control access to `*.com`, and send requests to _www.cnn.com_ and _www.abc.com_, the Istio gateway would not know which
-IP address to forward the requests to.
-This limitation is due to the limitation of [Envoy](https://www.envoyproxy.io), the proxy Istio is based on. Envoy
-routes traffic either to predefined hosts, or to predefined IP addresses, or to the original destination IP address of
-the request. In the case of the gateway the original destination IP of the request is lost (since the request was routed
-to the egress gateway and its destination IP address is the IP address of the gateway).
+> Note that this is only a problem for encrypted (HTTPS/TLS) requests and not simple HTTP where the `X-Forwarded-Host`
```

Comment:
The question is if Envoy knows to perform routing by the `X-Forwarded-Host`, or it can only report it.

Reply:
Wishful thinking I guess :-) I just thought we should at least mention HTTP, even thought the example is HTTPS using SNI. How about a note something like this:

> For simple HTTP traffic, a similar forward proxy would use the `X-Forwarded-Host` header for this purpose.